desktop/npm-helper.js

```diff
@@ -75,7 +75,7 @@ const commands = {
     help: 'Start electron with no hot reloading',
   },
   'build-dev': {
-    env: {NO_SERVER: 'true', DEBUG: 'express:*'},
```

Comment:
we don't need these express debug things anymore. webpack is stable and we don't need to see the individual requests